_pages/location.md

```diff
@@ -0,0 +1,319 @@
+    <li><a href="https://www.thefishmarketnewhaven.co.uk/">The Fishmarket</a> - a little 
+    further away on the coast in Newhaven but well worth a visit for the freshest possible 
+    fish & chips, sourced from the excellent 
+    <a href="https://www.welchfishmongers.com">Welch Fishmongers</a> next door.</li>
+    </ul>  
+  </p>
+  <h3>Burgers</h3>
+    <p>
+    You'll find a few branches of burger chains such as <a href="https://www.google.com/maps/place/fountain+park,+Dundee+St,+Edinburgh+EH11+1AF,+UK/@55.9414843,-3.2192654,17z/data=!3m1!4b1!4m5!3m4!1s0x4887c7a6567bf729:0x58fdfbfbb24837d0!8m2!3d55.9416438!4d-3.2168058">Five Guys</a> around Edinburgh, 
+    but the real place to go is <a href="http://www.breadmeatsbread.com/">Bread Meats 
+    Bread</a>, a Glasgow institution that came to Edinburgh a few years ago and is 
+    thriving here.
+    </p>
+  <h3>Restaurants</h3>
+  <p>
+  The Assembly Rooms’ location on George Street means that you’re not short of nearby 
+  options for good food.  On George Street itself you’ll find restaurants providing a 
+  variety of cuisine, including some well known names such as 
+  <a href="https://www.browns-restaurants.co.uk/restaurants/scotlandandnorthernireland/edinburgh/">Browns</a>, local tapas chain 
+  <a href="https://www.cafeandaluz.com/">Cafe Andaluz</a>, and more.  Just off George Street, some more hidden favourites include 
+  <a href="https://www.hakatayauk.com/">Hakataya</a>, for the best tonkatsu ramen in town.
+  </p>
+  <p>
+  Websites such as <a href="https://www.5pm.co.uk/">5pm</a> 
+  and <a href="https://www.opentable.co.uk/">opentable</a> can be useful for last minute 
+  table reservations.
+  </p>
+  <h3>Pubs and Bars</h3>
+  <p>
+  An evening in the pub with a beer or a whisky is a favourite pastime in Edinburgh, 
+  although in recent years gin has become an increasingly popular export! As such, 
+  Edinburgh has many quality pubs and bars, most of which will have a range of excellent 
+  single-malt whiskies. Many of the pubs around Edinburgh also serve good food! 
+  </p>
+  <p>
+  George Street is well placed for several good cocktail bars; 
+  <a href="https://www.bramblebar.co.uk/">Bramble</a>, 
+  <a href="http://pandaandsons.com/">Panda & Sons</a>, 
+  <a href="http://hoottheredeemer.com/">Hoot the Redeemer</a>, 
+  <a href="https://www.tigerlilyedinburgh.co.uk/">Tiger Lily</a> and the 
+  <a href="https://www.candybaredinburgh.co.uk/">Candy Bar</a> to name a few.  
+  </p>
+  <p>
+  Other good pubs nearby are the 
+  <a href="https://theabbotsford.com/">Abbotsford</a> for more traditional beer styles, 
+  <a href="https://www.fiercebeer.com/bars/edinburgh-bar">Fierce Bar</a> and 
+  <a href="http://thehangingbat.com/">The Hanging Bat</a> for more modern craft beer. 
+  <a href="http://www.redsquirreledinburgh.co.uk/">Red Squirrel</a> and 
+  <a href="https://www.brewdog.com/bars/uk/lothian-road">Brewdog</a> have good pub food menus. A little further away, 
```

Comment:
the brewdog link no longer works. This does:  https://www.brewdog.com/uk/brewdog-lothian-road